cmd/pipeline/bootstrap.yaml

```diff
@@ -49,7 +49,6 @@ edgepipeline_register:
   cluster_access_key: "${CLUSTER_ACCESS_KEY}"
   allowed_sources: "cdp-,fdp-"
   cluster_access_key_etcd_prefix: "${EDGE_PIPELINE_CLUSTER_ACCESS_KEY_ETCD_PREFIX:/devops/pipeline/v2/edge/cluster-access-key}"
-edgepipeline: {}
```

Comment:
why remove edgepipeline provider?

Reply:
edgepipeline provider should be removed to resolve circular dependencies